front/app/containers/Admin/messaging/texting/components/SMSCampaignForm.tsx

```diff
@@ -134,7 +134,10 @@ const SMSCampaignForm = ({
   const messageIsPastCharacterLimit = remainingChars < 0;
   const hasPhoneNumbersError = !hasPhoneNumbers || hasInvalidPhoneNumbersError;
   const hasMessageError = messageIsPastCharacterLimit;
-  const isButtonDisabled = hasMessageError || hasPhoneNumbersError;
+  const messageIsEmpty =
+    isNilOrError(inputMessage) || inputMessage.length === 0;
+  const isButtonDisabled =
+    hasMessageError || hasPhoneNumbersError || messageIsEmpty;
```

Comment:
I would classify `messageIsEmpty` as part of `hasMessageError` (at least that was the intention behind `hasMessageError`, to group all message-related errors):

```
const messageIsEmpty =
    isNilOrError(inputMessage) || inputMessage.length === 0;
const hasMessageError = messageIsPastCharacterLimit | messageIsEmpty;
const isButtonDisabled =
    hasMessageError || hasPhoneNumbersError;
```